packages/constellation/src/components/Base/Tag/Tag.types.ts

```diff
@@ -0,0 +1,30 @@
+import { CSSProperties, FC } from 'react'
+
+export type TagVariants = 'default' | 'alt' | 'success' | 'warning' | 'error'
+export type TagSizes = 'default' | 'large'
+
+export type Props = {
+  as?: 'div' | 'button' | 'a'
+
+  /**
+   * Text content of tag
+   */
+  content: string
+
+  /**
+   * The size of the tag, from a range of variants.
+   */
+  size?: TagSizes
+
+  /**
+   * Style object to be applied to the tag.
+   */
+  style?: CSSProperties
```

Comment:
Same question as in the layout PR -- are we sure we want to allow a general style prop? Beyond Blockchain's Content Security Policy for inline styles, I'm also concerned that the style prop could get misused/abused in terms of non-standard styling that deviates from the standards set by the styleguide + component library.

Reply:
That is a very good point, this component probably does not need it. I was thinking about what happens when you want to use a component like this and need to add a margin around it. I suppose that is what the padding component is for though.